.gitignore

```diff
@@ -7,4 +7,6 @@ war
 .gradle
 installer/appEngineSDK.zip
 gradle.properties
-logs
\ No newline at end of file
+logs
+pg_data
```

Comment:
I propose `pgdata` because that's what it's called in Postgres docs. https://www.postgresql.org/docs/9.1/static/storage-file-layout.html